### ISSUE TITLE
ykman oath reset throws python errors

### ISSUE THREAD

**Issue by Zytekaron** [Yubico/yubikey-manager#528]
- YubiKey Manager (ykman) version: 5.0.0
- How was it installed?: homebrew
- Operating system and version: macOS Ventura 13.0.1 (22A400)
- YubiKey model and version: 5C Nano (firmware 5.4.3)
- Bug description summary: python error when attempting to reset oath

May be related to #524

Steps to reproduce
---
```
S ykman oath reset         
```

Actual results and logs
---
```
ERROR: An unexpected error has occuredTraceback (most recent call last):
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/ykman/_cli/__main__.py", line 364, in main
    cli(obj={})
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/opt/homebrew/Cellar/ykman/5.0.0/libexec/lib/python3.10/site-packages/ykman/_cli/oath.py", line 119, in reset
    force or click.confirm(
TypeError: confirm() got an unexpected keyword argument 'prompt'
```

**Comment by dainnilsson** [comment]
Thanks for the report. Yes, this is the same underlying problem as #524 and there is already a fix in `main`. Once 5.0.1 is released this will be working again. In the meantime, it is possible to use the `--force` flag to bypass this failing confirmation prompt and reset the application.

**Comment by dainnilsson** [comment]
Now fixed in 5.0.1.
